message-analyzer/PostWithSentiment.cs

```diff
@@ -0,0 +1,23 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace MessageAnalyzer
+{
+    internal class PostWithSentiment : Post
```

Comment:
Can we get the better naming?

Reply:
What about SocialMediaMessageWithSentiment?  What do you suggest?

Reply:
@artursouza can give more input on naming :)

Reply:
I would have a single class and have the sentiment be null. I don't think it needs to have a separate class per step of the pipeline.

Reply:
Renamed SocialMediaPost and combined into 1 class.